lib/slack-ruby-bot/rspec/support/slack-ruby-bot/respond_with_error.rb

```diff
@@ -2,12 +2,20 @@
 
 RSpec::Matchers.define :respond_with_error do |error, error_message|
   match do |actual|
-    channel, user, message = parse(actual)
     allow(Giphy).to receive(:random)
+
+    client = if respond_to?(:client)
```

Comment:
I had a hard rubocop assisted transition away from one liners, don't break my therapy man :)

tl;dr; this just reads nicer, after a while I've started to agree